crates/sats/src/relation.rs

```diff
@@ -147,15 +151,15 @@ pub struct ColumnOnlyField<'a> {
 // TODO(perf): Remove `Clone` derivation.
 #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct Column {
-    pub field: FieldName,
+    pub field: Arc<FieldName>,
```

Comment:
I actually think this doesn't need to be an Arc, for reasons I will explain below.